src/tests/modules/oid.rs

```diff
@@ -27,6 +27,7 @@ fn byte_string_oid() {
 }
 
 #[test]
+#[allow(clippy::eq_op)]
```

Comment:
[eq_op](https://rust-lang.github.io/rust-clippy/master/index.html#eq_op) produces a warning when something is compared to itself, but we need to do that in this test.